hirundo/cli.py

```diff
@@ -32,18 +37,30 @@ def upsert_env(var_name: str, var_value: str):
         var_name: The name of the environment variable to change.
         var_value: The new value of the environment variable.
     """
-    dotenv = "./.env"
     regex = re.compile(rf"^{var_name}=.*$")
-    with open(dotenv) as f:
-        lines = f.readlines()
+    lines = []
+    if os.path.exists(dotenv_filepath):
```

Comment:
Why the `if` here? What happens when it's `False`?

Reply:
The lines will stay an empty list and then the file will be created